test/error_tracker/integrations/oban_test.exs

```diff
@@ -0,0 +1,72 @@
+defmodule ErrorTracker.Integrations.ObanTest do
+  use ErrorTracker.Test.Case
+
+  alias ErrorTracker.Error
+  alias ErrorTracker.Occurrence
+
+  setup do
+    attach_telemetry()
+
+    :ok
+  end
+
+  test "attaches to Oban events" do
+    assert attached?([:oban, :job, :exception])
+  end
```

Comment:
I'm not sure if we need to test the attachment itself, as it is a detail of the implementation - it will be implicitly tested when we test that the error is stored.